packages/search/src/features/records/service.ts

```diff
@@ -1027,36 +1019,3 @@ export async function getRecordById<T extends Array<keyof StateIdenfitiers>>(
 
   return record as StateIdenfitiers[T[number]]
 }
-
-function resolveReferenceFullUrls(bundle: Bundle, entries: BundleEntry[]) {
-  return entries.map((entry) => {
-    const resource = entry.resource!
-    if (isComposition(resource)) {
-      resource.section?.forEach((section) => {
-        section.entry?.forEach((entry) => {
-          entry.reference = getFromBundleById(
-            bundle,
-            entry.reference!.split('/')[1]
-          )?.fullUrl
-        })
-      })
-    }
-    if (isEncounter(resource)) {
-      resource.location?.forEach(({ location }) => {
-        location.reference = getFromBundleById(
-          bundle,
-          location.reference.split('/')[1]
-        )?.fullUrl
-      })
-    }
-
-    if (isRelatedPerson(resource) && resource.patient?.reference) {
-      resource.patient.reference = getFromBundleById(
-        bundle,
-        resource.patient.reference.split('/')[1]
-      ).fullUrl
-    }
```

Comment:
Aren't these still needed? I remembered Hearth does transform some references to the URL form. I might be mistaken though 🤔